app/controllers/points_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PointsController < ApplicationController
+  include Secured
+  include ExistingUser
+
+  def index
+    points_valid_member
+  end
+
+  def events
+    points_valid_member
+  end
+
+  def manual_points
+    points_valid_member
+  end
+
+  def accomplishments
+    points_valid_member
+  end
+
+  def points_valid_member
+    @member = Member.find_by_uid(session[:app_user]['uid'])
+    unless @member.paid_dues?
+      flash[:alert] = 'You must pay dues to see your points.'
+      redirect_to('/dashboard')
+    end
+  end
```

Comment:
I see you added an index and points valid! Nice thinking on making sue their dues have been paid